api/handlers/btcPayServer.js

```diff
@@ -0,0 +1,54 @@
+const axios = require('axios')
+const {
+    BTCPAYSERVER
+} = require('../config/credentials')
+
+const BTCPayServerAPIKey = BTCPAYSERVER.API_KEY;
+const BTCPayServerStoreId = BTCPAYSERVER.STORE_ID;
```

Comment:
Rename to `BTCPAYSERVER_API_KEY` and `BTCPAYSERVER_STORE_ID` for constants like this